test/studies/shootout/fasta/kbrady/fasta-qio.chpl

```diff
@@ -158,18 +151,19 @@ proc repeatMake(desc : string, alu : [], n : int) {
   stdout.writef("%s", desc);
   var r : int = alu.size;
   var s : [0..(r+LINE_LENGTH)] int(8);
-  for indx in s.domain {
-    s[indx] = alu[indx % r];
-  }
+  for d in s.domain do 
+    s[d] = alu[d % r];
+  
   var j : int;
-
   for i in 0..(n / LINE_LENGTH)-1 {
     j = (i * LINE_LENGTH) % r;
     stdout.write(s[j..#LINE_LENGTH], 10:int(8));
   }
-  if (n % LINE_LENGTH) {
-    j = ((n / LINE_LENGTH)*LINE_LENGTH) % r;
-    stdout.write(s[j..#(n % LINE_LENGTH)], 10:int(8));
+
+  var m = n % LINE_LENGTH;
```

Comment:
Super nit-picky: make 'm' const, and maybe give it a better name like 'trail', because I think this section is trying to write the last and possibly shorter line.